sdk/eventhub/event-hubs/src/sender.ts

```diff
@@ -58,6 +59,33 @@ export class EventHubProducer {
     this._eventHubSender = EventHubSender.create(this._context, partitionId);
   }
 
+  /**
+   * Creates a batch where event data objects can be added for later `Send` call
+   * @param options  Options to define partition key and max message size.
+   * @returns Promise<EventDataBatch>
+   */
+  async createBatch(options?: BatchOptions): Promise<EventDataBatch> {
+    if (!options) {
+      options = {};
+    }
+    let maxMessageSize = await this._eventHubSender!.getMaxMessageSize();
```

Comment:
Will this throw an error if the producer has already been closed?

Reply:
Nope, it will create the sender link.

@ShivangiReja We should call `this._throwIfSenderOrConnectionClosed();` as the first step in `createBatch()` to avoid this